install/ansible/istio/tasks/install_addons.yml

```diff
@@ -20,4 +20,22 @@
   when:
     - is_istioaddon_iterable
     - "'jaeger' in istio.addon"
-    - "cluster_flavour == 'k8s'"
\ No newline at end of file
+    - "cluster_flavour == 'k8s'"
+
+- name: Install Kiali on Openshift
+  shell: |
+    curl https://raw.githubusercontent.com/kiali/kiali/master/deploy/openshift/kiali-configmap.yaml | VERSION_LABEL=master envsubst | {{ cmd_path }} create -n istio-system -f -
+    curl https://raw.githubusercontent.com/kiali/kiali/master/deploy/openshift/kiali.yaml | IMAGE_NAME=kiali/kiali IMAGE_VERSION=latest NAMESPACE=istio-system VERSION_LABEL=master VERBOSE_MODE=4 envsubst | {{ cmd_path }} create -n istio-system -f -
+  when:
+    - is_istioaddon_iterable
+    - "'kiali' in istio.addon"
+    - "cluster_flavour == 'ocp'"
+
+- name: Install Kiali on Kubernetes
+  shell: |
+   curl https://raw.githubusercontent.com/kiali/kiali/master/deploy/kubernetes/kiali-configmap.yaml | VERSION_LABEL=master envsubst | {{ cmd_path }} create -n istio-system -f -
+   curl https://raw.githubusercontent.com/kiali/kiali/master/deploy/kubernetes/kiali.yaml | IMAGE_NAME=kiali/kiali IMAGE_VERSION=latest NAMESPACE=istio-system VERSION_LABEL=master  VERBOSE_MODE=4 envsubst | {{ cmd_path }} create -n istio-system -f -
```

Comment:
the `latest` may not work sometime, can we use a released version here?

Reply:
we can use released version.

do you have something version in mind?

Reply:
@gyliu513  set version to 0.3.0.Alpha.